### DISCUSSION TITLE
Laravel Nova 4 with nova-permission package got error: View [app] not found

### DISCUSSION THREAD

**Discussion by gofunz** [laravel/nova-issues#4068] [Q&A]
- Laravel Version: 9.9.0
- Nova Version: 4.2.3
- PHP Version: 8.1.4
- Database Driver & Version: Mysql & 8.0.28
- Operating System and Version: Laravel Sail 1.13.10
- Browser type and version: Chrome 100.0.4896.127
- Reproduction Repository: https://github.com/gogoafuzzz/nova-admin-panel
- Flare Detail: https://flareapp.io/share/47q1WJA7#F42

### Description:
I installed nova 4 and [kiritokatklian/nova-permission](https://github.com/kiritokatklian/nova-permission) package, i got this error.

But I saw some related issues and I'm not sure if it's a Laravel Nova 4 issue

- https://github.com/laravel/nova-issues/issues/3851
- https://github.com/laravel/nova-issues/issues/3867

### Detailed steps to reproduce the issue on a fresh Nova installation:
```shell

git clone git@github.com:gogoafuzzz/nova-admin-panel.git

cp .env.example .env

composer install

sail up -d

sail artisan migrate
```

**Comment by crynobone** [comment]
https://github.com/gogoafuzzz/nova-admin-panel/blob/main/config/nova.php missing correct `nova` configuration.

**Comment by maximegaul** [comment]
@gofunz i guess you forgot this part of the migration process, about updating the config/nova.php file ?

https://nova.laravel.com/docs/upgrade.html#updating-configuration-assets-and-translations
